lib/collection/src/operations/conversions.rs

```diff
@@ -1435,6 +1435,24 @@ impl From<ReshardingDirection> for api::grpc::qdrant::ReshardingDirection {
     }
 }
 
+impl From<CollectionShardKeys> for Vec<api::grpc::qdrant::ShardKey> {
+    fn from(value: CollectionShardKeys) -> Self {
+        value
+            .shard_keys
+            .into_iter()
+            .map(|k| {
+                let key = match k {
+                    ShardKey::Keyword(k) => {
+                        api::grpc::qdrant::shard_key::Key::Keyword(k.to_string())
+                    }
+                    ShardKey::Number(n) => api::grpc::qdrant::shard_key::Key::Number(n),
+                };
+                api::grpc::qdrant::ShardKey { key: Some(key) }
+            })
+            .collect()
+    }
+}
```

Comment:
nit: There should be a direct conversion from `segment::types::ShardKey` to `api::grpc::qdrant::ShardKey`, and then use it here.